### ISSUE TITLE
import -p=<n> doesn't work

### ISSUE THREAD

**Issue by ron-wolf** [climech/grit#19]
Importing with a specified predecessor just creates a new root node no matter what.

```fish
> grit stat 1
(1) ──── (2)

ID: 1
Name: 2021-04-04
Status: inactive (0/5)
Parents: 0
Children: 1
Created: 2021-04-04 15:10:15
```
```fish
> grit import -p=1 (cat | psub)
Test node
^D
[ ] Test node (12)
Imported 1 trees (1 nodes)
```
```fish
> grit stat 12
ID: 12
Name: Test node
Status: inactive (0/1)
Parents: 0
Children: 0
Created: 2021-04-04 16:07:51
```

**Comment by ron-wolf** [comment]
Maybe I’m misunderstanding how the “import” subcommand is supposed to work. Is there a feature that currently matches what I’m trying to do here, or does “import” require a more specific input format?

**Comment by climech** [comment]
Yep, the command was broken, and got included in the release by mistake, see: #1.
